main.tf

```
@@ -223,7 +223,7 @@ resource "aws_batch_job_queue" "this" {
   name                  = each.value.name
   state                 = each.value.state
   priority              = each.value.priority
-  scheduling_policy_arn = try(each.value.scheduling_policy_arn, aws_batch_scheduling_policy.this[each.key].arn)
```

we should be able to just add a `null` at the end here and that should solve this (I think)

```suggestion
  scheduling_policy_arn = try(each.value.scheduling_policy_arn, aws_batch_scheduling_policy.this[each.key].arn, null)
```

---

@bryantbiggs the issue is that aws_batch_scheduling_policy is auto-created(lines 236-255), so Terraform will still attach the policy.